talkideas.md

```diff
@@ -8,3 +8,39 @@ In-depth debugger talk
 ======================
 *TODO: Rewrite*
 
+
+Developing Developers
+=====================
+Software development is a young profession. Our career paths are still geared up to accept junior developers from academia, make them productive in a team, and then preparing them for a career in management. Technical skills and experience are devalued in preference of line management and project management.
+
+We have a responsibility for developing the technical skills and experience of developers, for building a better profession and for creating a healthy environment to work in. How can we shape careers that challenge and value developers, and how do you identify and respond when things go wrong?
+
+This talk will use my insights from building career paths from scratch at emerging companies, and examine the successes and failures that are common across the industry.
+
+
+Modular CSS with WebPack
+========================
+Composition has long been a goal of CSS design. Creating a rich library of styles that can be reused and combined saves time in the long run, but often lead to unmaintainable or verbose code.
+
+WebPack is commonly used for bundling JS modules, but its flexibility means it can also be used to create modular stylesheets: local styles that can be imported and exported just like scripts.
+
+This talk introduces WebPack and will demonstrate how to write, bundle and use CSS modules. We'll also examine the benefits of using WebPack, and why it can help in ways other tools have failed in the past.
+
+
+Dispelling Devops with Ansible
+==============================
+Do you fear that owning your own web server is reserved for the elite and all-powerful society of sysadmins? Then fear not, the fact is that it's never been easier.
+
+Ansible is a powerful tool that lets you control servers by configuration instead of command. It's designed for ease-of-use, and this talk will show how installing software, automating tasks and setting up access control is managed using its simple YAML interface. We'll use these lessons to set up a basic website on a fresh server, and explain some of the fundamentals of hosting a site along the way.
+
+- This talk is aimed at hosting beginners.
+- The talk will features some command-line usage, but user-friendly tools will be the major theme.
+- I want to demonstrate Ansible alongside other tools designed for ease-of-use, such as Digital Ocean and the Caddy webserver.
+
```

Comment:
This also looks good. I'd maybe tweak the title somewhat, it sounds like you're trying to put devops out of a job. Maybe Dispelling DevOps Myths or something?

I do enjoy live demo style talks, but I'd also be wary of conference wifi and have some fallbacks in case the site/DO/something isn't working.